velox/functions/prestosql/BinaryFunctions.h

```diff
@@ -415,9 +415,10 @@ struct ToIEEE754Bits64 {
   FOLLY_ALWAYS_INLINE void call(
       out_type<Varbinary>& result,
       const arg_type<double>& input) {
-    result.resize(64);
-    bits::toString((const uint64_t*)&input, 0, 64, result.data());
-    std::reverse(result.data(), result.data() + 64);
+    static constexpr auto kTypeLength = sizeof(int64_t);
```

Comment:
This code is exactly the same as ToBigEndian64 as above. I would have thought that you have to memcpy the double into a bit array and then do a reverse like the previous code.